src/Stays/Bookings/Bookings.ts

```diff
@@ -0,0 +1,62 @@
+import { Client } from 'Client'
+import { StaysBooking } from 'Stays/types'
+import { Resource } from '../../Resource'
+import { DuffelResponse } from '../../types'
+
+export interface StaysBookingPayload {
+  quote_id: string
+  guests: Array<{
+    given_name: string
+    family_name: string
+    born_on: string
+  }>
+  email: string
+  phone_number: string
+  accommodation_special_requests?: string
+}
+
+export class StaysBookings extends Resource {
```

Comment:
Interesting, don't we have the concept of namespace in Javascript?

I mean, why can't be just called `Bookings`?

Then importing it from somewhere I could alias anyway I want, right?

```
import { Bookings as Blebs } from "stays/bookings";
```

Reply:
Yes. I just wanted to be explicit here as we also have a whole `bookings` folder that's just flights stuff. We are also exposing this as `duffel.staysBookings` or `duffel.staysQuotes` rather than `duffel.bookings` or `duffel.quotes` which could be ambiguous. 

To be fair, we are probably due to some amount of reorganisations here with the introduction of Stays (e.g. it's probably high time we have `duffel.flights.*` and `duffel.stays.*` but such reorganisation would be a breaking change and probably would require a bit more thoughts, so I'm opted for this for now.

Reply:
Ahh I see. But if we add `duffel.stays.*` now, and we refactor `duffel.flights.*` later, it wouldn't be a breaking change, right? It will be inconsistent for a start, but step forward to the direction that we want. Would that be possible?

Reply:
Good point. Yeah that's something we can definitely do. Will make an update

Reply:
updated!